test/stream_compression.jl

```diff
@@ -15,15 +16,15 @@ erat ex bibendum ipsum, sed varius ipsum ipsum vitae dui.
     @test LZ4CompressorStream <: TranscodingStream
     @test LZ4DecompressorStream <: TranscodingStream
 
-    compressed = transcode(LZ4Compressor, text)
+    compressed = transcode(LZ4Compressor, Vector{UInt8}(text))
```

Comment:
Make a note of https://github.com/bicycle1885/TranscodingStreams.jl/issues/39 in these so we change this once that issue is closed.